data/json/items/comestibles/brewing.json

```
@@ -374,7 +374,7 @@
     "id": "brew_vinegar",
     "name": "unfermented vinegar",
     "description": "Mixture of water, alcohol and fruit juice that will eventually become vinegar.",
-    "weight": "127 g",
+    "weight": "1016 g",
```

```suggestion
    "weight": "15 g",
```
This is the weight per charge, according to `JSON_INFO.md`.